ReceiptStorage.h

```diff
@@ -0,0 +1,20 @@
+#ifndef RECEIPTSTORAGE_H_INCLUDED
+#define RECEIPTSTORAGE_H_INCLUDED
+const int CAPACITY=100;

```

Comment:
Avoid declaring anything outside the class declaration, it keeps the files tidy. In this case, our capacity will have to be dynamic, and the _vector_ class has a _vector::size()_ method, which returns what the _receiptSize_ and _offerSize_ are holding.